### DISCUSSION TITLE
принудительно включен rx-checksum offload, badsum пакеты

### DISCUSSION THREAD

**Discussion by cicsaw** [bol-van/zapret#411] [General]
Хочу вырубить эту бяку и пересобрать прошивку OpenWrt 23.05.4 для ZyXEL Keenetic Lite III rev A • CPU: MediaTek MT7620N (MIPS24Kc).
Я так понял нужно рыть в ...target/linux/ramips/mt7620 и править конфиг. Только вот что именно ?

**Comment by bol-van** [comment]
На MT7621   tplink C6U  режет только при приеме badsum. На отсылку не режет, то есть с самого роутера работает, за ним из LAN - нет
